docs/source/guide/auth_setup.md

```diff
@@ -100,86 +98,10 @@ The details will vary depending on your IdP, but in general you will complete th
 
-# Specify organization to assign it to all users on the platform
-# Warning: the user with this email must be registered before any LDAP users log in 
-AUTH_LDAP_ORGANIZATION_OWNER_EMAIL=heartex@heartex.net
+## Manage user access only with SSO 
 
-# Populate the user from the LDAP directory:
-# firstName, lastName, mail, sAMAccountName are taken from your LDAP record 
-AUTH_LDAP_USER_ATTR_MAP_FIRST_NAME=firstName
-AUTH_LDAP_USER_ATTR_MAP_LAST_NAME=lastName
-# Specify the field to use for AUTH_LDAP_USER_QUERY_FIELD as 'email'
-AUTH_LDAP_USER_ATTR_MAP_EMAIL=mail
-# Specify the field to use for AUTH_LDAP_USER_QUERY_FIELD as 'username' 
-AUTH_LDAP_USER_ATTR_MAP_USERNAME=sAMAccountName
-
-# Query the authenticating user in Label Studio, it can be [email|username]
-AUTH_LDAP_USER_QUERY_FIELD=email
-
-# Group parameters
-AUTH_LDAP_GROUP_SEARCH_BASE_DN=ou=Users,o=group-id,dc=example,dc=com
-AUTH_LDAP_GROUP_SEARCH_FILTER_STR=(objectClass=groupOfNames)
-AUTH_LDAP_GROUP_TYPE=ou
-```
-
-If you want to use a recursive scan to search in several LDAP groups to grant access to Label Studio, instead of relying on the simple search used by `AUTH_LDAP_USER_ON_TEMPLATE`, update your environment variables file like the following:
-```bash
-# Leave this parameter empty
-AUTH_LDAP_USER_DN_TEMPLATE=""
-# Specify the groups that you want to be able to log in, separated by ';' 
-AUTH_LDAP_USER_SEARCH_BASES="ou=guests,dc=domain,dc=com;ou=owners,dc=domain,dc=com"
-```
-
-
-## User Role Mappings
-
-Map LDAP groups to specific Label Studio Enterprise roles using `;` to specify several groups.
-
-```bash
-AUTH_LDAP_ORGANIZATION_ROLE_ADMINISTRATOR=cn=admins,ou=users,o=123abc,dc=example,dc=com 
-AUTH_LDAP_ORGANIZATION_ROLE_MANAGER=cn=managers,ou=users,o=123abc,dc=example,dc=com 
-AUTH_LDAP_ORGANIZATION_ROLE_REVIEWER=cn=reviewers,ou=users,o=123abc,dc=example,dc=com
-AUTH_LDAP_ORGANIZATION_ROLE_ANNOTATOR=cn=annotators,ou=users,o=123abc,dc=example,dc=com;cn=guests,ou=users,o=123abc,dc=example,dc=com
-AUTH_LDAP_ORGANIZATION_ROLE_NOT_ACTIVATED=cn=not,ou=users,o=123abc,dc=example,dc=com 
-AUTH_LDAP_ORGANIZATION_ROLE_DEACTIVATED=cn=deactivated,ou=users,o=123abc,dc=example,dc=com
-```
-
-
-## Workspace Mappings
-
-Map LDAP groups to specific Label Studio workspaces. Use a JSON format where keys are workspace titles and values are LDAP groups. Split groups with `;` to specify several groups. 
-
-```bash
-AUTH_LDAP_ORGANIZATION_WORKSPACES='{"Workspace 1":"cn=team1,ou=users,o=60cbc901ec2e8e387a3b2d3e,dc=jumpcloud,dc=com","Workspace 2":"cn=team2,ou=users,o=60cbc901ec2e8e387a3b2d3e,dc=jumpcloud,dc=com"}'
-```
-
-
-## Manage user access only with LDAP or SSO 
-
-If you want to manage Label Studio roles and workspaces entirely with LDAP or single sign-on (SSO), add the following to your environment variable file:
+If you want to manage Label Studio roles and workspaces entirely with single sign-on (SSO), add the following to your environment variable file:
```

Comment:
btw, let's add here to "Manage user access only with SSO", that if a customer uses SaaS, he can ask us to enable this flags for his organization (we can do it via billing settings). Also we should mention here, that we can disable "Common login" by their request and users must use SSO login only. Common login will be completely disabled.